docs/guides/get-started-with-primitives.mdx

```diff
@@ -264,31 +264,79 @@ print(f"  > Metadata: {result.metadata[0]}")
+or the legacy V1 implementation, [`qiskit.primitives.BackendSampler`](/api/qiskit/qiskit.primitives.BackendSampler). 
+Likewise, the `Estimator` primitive can be run with any provider using [`qiskit.primitives.BackendEstimatorV2`](../api/qiskit/qiskit.primitives.BackendEstimatorV2), 
+or the legacy V1 implementation, [`qiskit.primitives.BackendEstimator`](/api/qiskit/qiskit.primitives.BackendEstimator).
 
 Some providers implement primitives natively (see [the Qiskit Ecosystem page](https://qiskit.github.io/ecosystem#provider) for more details).
 
 ### Example: BackendEstimator
 
+<Tabs>
+  <TabItem value="BackendEstimatorV2" label="BackendEstimator V2">
 ```python
+from qiskit.primitives import BackendEstimatorV2
+from <some_qiskit_provider> import QiskitProvider
+
+provider = QiskitProvider()
+backend = provider.get_backend('backend_name')
+estimator = BackendEstimatorV2(backend)
+```
+  </TabItem>
+
+  <TabItem value="BackendEstimatorV1" label="BackendEstimator (V1)">
+  ```python
 from qiskit.primitives import BackendEstimator
 from <some_qiskit_provider> import QiskitProvider
 
 provider = QiskitProvider()
 backend = provider.get_backend('backend_name')
 estimator = BackendEstimator(backend)
 ```
+  </TabItem>
 
 ### Example: BackendSampler
 
+<Tabs>
+  <TabItem value="BackendSamplerV2" label="BackendSampler V2">
 ```python
+from qiskit.primitives import BackendSamplerV2
+from <some_qiskit_provider> import QiskitProvider
+
+provider = QiskitProvider()
+backend = provider.get_backend('backend_name')
+sampler = BackendSamplerV2(backend)
+```
+  </TabItem>
+
+  <TabItem value="BackendSamplerV1" label="BackendSampler (V1)">
+  ```python
 from qiskit.primitives import BackendSampler
 from <some_qiskit_provider> import QiskitProvider
 
 provider = QiskitProvider()
 backend = provider.get_backend('backend_name')
 sampler = BackendSampler(backend)
 ```
+  </TabItem>
+
+### Similarities and differences between backend and Runtime primitives (V2)
+
+- The inputs to [`qiskit.primitives.BackendSamplerV2`](/api/qiskit/qiskit.primitives.BackendSamplerV2) and [`qiskit.primitives.BackendEstimatorV2`](../api/qiskit/qiskit.primitives.BackendEstimatorV2)
+follow the same PUB format as the primitives in Qiskit Runtime, so do the outputs, see (link to https://docs.quantum.ibm.com/guides/primitive-input-output). 
+However, there can be differences in the fields of the returned metadata.
+
+- The [`qiskit.primitives.BackendEstimatorV2`](/api/qiskit/qiskit.primitives.BackendEstimatorV2) class offers no measurement or gate error mitigation implementations out-of-the-box, as
+backend primitives are designed to run locally in the user's machine. 
+
+- The [`qiskit.primitives.BackendSamplerV2`](/api/qiskit/qiskit.primitives.BackendSamplerV2) class requires a backend that supports the `memory` option.
+
+- The backend primitives V2 interfaces expose custom `Sampler` and `Estimator` `Options` for that are different to the Runtime implementations. 
```

Comment:
I wanted to link to the specific option classes but I see that they are not listed in the docs, which might be by design, not sure. I am referring to:

- BackendEstimatorV2 options class: https://github.com/Qiskit/qiskit/blob/main/qiskit/primitives/backend_estimator_v2.py#L41-L57
- BackendSamplerV2 options class: https://github.com/Qiskit/qiskit/blob/main/qiskit/primitives/backend_sampler_v2.py#L43-L54

Getting into details about the options in this page seems to be a bit too much (as it is a get-started-with page) that's why I thought linking to the API docs would be a good solution. I guess we could also omit this item.

Reply:
That seems like an error.  Investigating.

Reply:
I tracked this down.  Those classes are not public, so all we can do is point them to the BackendEstimatorV2 class docs and the BackendSamplerV2 class docs

Reply:
It might be unintentional that they are not documented, if someone forgot to list them in `qiskit/primitives/__init__.py` docstring. Consider asking in the Qiskit dev Slack what should be done

Reply:
> It might be unintentional that they are not documented, if someone forgot to list them in `qiskit/primitives/__init__.py` docstring. Consider asking in the Qiskit dev Slack what should be done

It's intentional.  They're not documented because they are not public.  I was told that it would take a little more work to make them public because the name would be overloaded.  It's under consideration for the next release.  For now, we just have to link to the docs I mentioned in my previous comment.